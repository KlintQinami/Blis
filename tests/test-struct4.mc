struct foo {
    int a;
    float b;
};

int main() {
    foo temp = foo(42, 1337.0);
    print(temp.a);
    printf(temp.b);

    return 0;   
}
