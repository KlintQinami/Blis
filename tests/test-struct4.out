42
1337.000000
